sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithValueStateTTLSuite.scala

```diff
@@ -399,7 +205,7 @@ class TransformWithValueStateTTLSuite
         AddData(inputStream, InputEvent(ttlKey, "get_ttl_value_from_state", -1)),
         AddData(inputStream, InputEvent(noTtlKey, "get_ttl_value_from_state", -1)),
         AdvanceManualClock(1 * 1000),
-        CheckNewAnswer(OutputEvent(ttlKey, -1, isTTLValue = true, 61000)),
+        CheckNewAnswer(OutputEvent(ttlKey, 1, isTTLValue = true, 61000)),
```

Comment:
It wasn't incorrect before - I had changed the getTTLValueFromState function to return the associated value with the TTL for ListState, and had to change it for ValueState so that the APIs would match. 
-1 was the placeholder value for when `isTTLValue=true`